packages/components/src/utils/json.ts

```diff
@@ -4,7 +4,7 @@ export const parseJSONAttribute = <T>(attribute: T | string): T => {
       JSON.parse(
         attribute
           .replace(/'/g, '"') // convert single quotes to double quotes
-          .replace(/[\s"]?([\w-]+)[\s"]?:/g, '"$1":') // wrap keys in double quotes
+          .replace(/[\s"]?([\w-]+)[\s"]?:([^//])/g, '"$1":$2') // wrap keys in double quotes if they don't have them but ignore potential urls
```

Comment:
I guess this can be reverted?

Reply:
Question is, should we? I think it is an improvement to before?

Reply:
What is it improving? It got more powerful and complex covering cases that don't exist.